TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/MovementServiceBinder.java

```diff
@@ -0,0 +1,60 @@
+package com.novoda.tpbot.bot;
+
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.os.IBinder;
+
+import com.novoda.tpbot.bot.device.DeviceConnection;
+
+class MovementServiceBinder {
```

Comment:
Still some work to be done around `Service`s, ideally we want to hide all of this from the `activity`so that the activity only interacts with a presenter.